pkg/updater/service/helper/main.go

```diff
@@ -36,6 +36,7 @@ type privilegeCommand struct {
 	Command string `json:"command,omitempty"`
 	Unit    string `json:"unit,omitempty"`
 	Path    string `json:"path,omitempty"`
+	Target  string `json:"target,omitempty"`
```

Comment:
nit: why not reuse path, or whats the difference between path and target?

Reply:
reused path